src/roadmap/releases.md

```diff
@@ -193,15 +205,23 @@ Change the `generate_params.toml` file in **each** new chart as follows:
   1. If there's a `[workflowManager]` section, change `versionsApiURL` to
      `"https://versions.deis.com"` and `doctorApiURL` to `"https://doctor.deis.com"`
 
-Change the `workflow-$WORKFLOW_RELEASE/tpl/deis-controller-deployment.yaml` file:
+Change the `workflow-$WORKFLOW_RELEASE/tpl/deis-controller-deployment.yaml` and
+`workflow-$WORKFLOW_RELEASE/tpl/deis-controller-rc.yaml` files:
```

Comment:
The RC shouldn't really need any change - It's a 0 pod dealio and doesn't boot pods in the same label selector space as Deployment